src/styles/components/_component-card.scss

```diff
@@ -6,13 +6,14 @@
 
   .component-card {
     margin: 0;
-    border: 2px solid black;
+    border: 2px solid var(--foreground-color);
```

Comment:
This is that border I think I mentioned to you. Right now, the border is white, but it should stay black (at least according to the designs) 

`border: 2px solid var(--background-color);`

Reply:
Got it!